tests/plugins/org.polarsys.capella.test.diagram.filters.ju/src/org/polarsys/capella/test/diagram/filters/ju/ldfb/HideDiagramTitleBlocksTest.java

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) 2020 THALES GLOBAL SERVICES.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *  
+ * Contributors:
+ *    Thales - initial API and implementation
+ *******************************************************************************/
+package org.polarsys.capella.test.diagram.filters.ju.ldfb;
+
+import java.util.Arrays;
+import java.util.List;
+import org.polarsys.capella.core.sirius.analysis.constants.IFilterNameConstants;
+import org.polarsys.capella.test.diagram.filters.ju.DiagramTitleBlockFilterTestCase;
+
+public class HideDiagramTitleBlocksTest extends DiagramTitleBlockFilterTestCase {

```

Comment:
Rename this tests by appending the diag tipe
ex HideDiagramTitleBlocksTest -> to -> HideDiagramTitleBlocksTest LAB
Also for all the files for other  added tests in this review (CDB, SAB, LAB etc)